docs/Cleanup.md

```diff
@@ -10,12 +10,9 @@ First we need to ensure our [environment variables](./Environment-Variables.md)
 
 ## Remove each resource group
 
-This command will delete all resources in *all* resource groups. Run
-with caution.
+This command will delete all resources in *all* resource groups. Run with caution.
 
-Note, that this command will not fully delete the resource group if
-you have Azure Backup enabled since the Recovery Services Vault will
-not be deleted (it's got the backups of you data!).
+Note, that this command will not fully delete the resource group if you have Azure Backup enabled since the Recovery Services Vault will not be deleted (it's got the backups of you data!).
 
 ``` bash
 for filename in $MOODLE_AZURE_WORKSPACE/*; do az group delete --yes --name $(basename $filename) --no-wait; done
```

Comment:
Still got a Moodle reference here (and likely in other places where this environment variables are used)